managers/views.py

```diff
@@ -1,8 +1,36 @@
+from django.db.models import Count
+from rest_framework import renderers
 from rest_framework import viewsets
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from users.models import CustomUser
 from managers.models import Project
 from managers.serializers import ProjectSerializer
 
 
 class ProjectViewSet(viewsets.ModelViewSet):
     queryset = Project.objects.all()
     serializer_class = ProjectSerializer
+
+
+class ProjectsList(APIView):
+    renderer_classes = [renderers.TemplateHTMLRenderer]
+    template_name = 'managers/projects_list.html'
```

Comment:
I didn't saw it at first, so I have dropped comment in the following pull request, so I will link it here https://github.com/Code-Poets/sheetstorm/pull/86#discussion_r264067210 .